version.gradle

```diff
@@ -76,7 +76,11 @@ task configureVersionInfo {
         jmeFullVersion = jmeMainVersion
         jmePomVersion  = jmeVersion
         
-        if (jmeBranchName != "master") {
+        if (jmeGitTag.startsWith(jmeMainVersion)) {
```

Comment:
Can you update the 4 examples in the top comment to match this new behavior? I want to see how it looks like now